### PR TITLE
feat: add helper functions for working with input/output of host functions as bytes

### PR DESCRIPTION
**Pull request by zshipko**

### CODE REVIEW

File: python/extism/extism.py [hunk 1]
```diff
@@ -464,6 +464,24 @@ def memory_at_offset(self, offs):
         len = _lib.extism_current_plugin_memory_length(self.pointer, offs)
         return Memory(offs, len)
 
+    def return_bytes(self, output, b: bytes):
+        mem = self.alloc(len(b))
+        self.memory(mem)[:] = b
+        output.value = mem.offset
```
Comment (bhelx) [inline python/extism/extism.py:470]: should this be `ouput[0] = `?

Comment (bhelx) [reply python/extism/extism.py:470]: Nevermind. I understand how it works now

**Review by bhelx — DISMISSED** [review]
Let's get this merged. I'll follow up with the types in python. I think we need them in a few more places.

**Comment by zshipko** [comment]
Oops, I didn't see that last comment and ended up adding the Python types. I will merge once CI passes.

**Pull request merged**